coalib/parsing/StringProcessing/Core.py

```diff
@@ -328,6 +328,38 @@ def escape(string, escape_chars, escape_with="\\"):
     return string
 
 
+def convert_to_raw(string, exceptions=""):
+    """
+    Converts a string to its raw form, converting all backslash to double
+    backslash except when the backslash escapes a character given in
+    exceptions.
+
+    :param string:     The given string that needs to be converted
+    :param exceptions: A list of characters that if escaped with backslash
+                       should not be converted to double backslash.
+    :return:           Returns the corresponding raw string.
+    """
+    exceptions += "/"
```

Comment:
Actually, on second thought, I think adding `/` to the list of exceptions would go better in the `LineParser: Convert keys and values to raw form` commit. This way we don't tamper with the function (in case somebody wants a `convert_to_raw` without forward slash?).